Tests/Functional/Resolver/RichtextAllowTagsResolverTest.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Functional\Resolver;
+
+use Nimut\TestingFramework\TestCase\FunctionalTestCase;
+use TYPO3\CMS\Core\Configuration\Loader\YamlFileLoader;
+use TYPO3\CMS\Core\Utility\GeneralUtility;
+use WebVision\WvDeepltranslate\Resolver\RichtextAllowTagsResolver;
+
+class RichtextAllowTagsResolverTest extends FunctionalTestCase
```

Comment:
New test class, let's use `final` here. I know it's missing in other test classes, but let's have it for the new one directly.